styles.css

```diff
@@ -914,4 +914,203 @@ ul {
   .hamburger.active .line:nth-child(3) {
     transform: translateY(-8px) rotate(-45deg);
   }
+
+  .desktop-only {
+    display: none;
+  }
+
+  .pop-up {
+    width: 70%;
+    top: 20%;
+    left: 30%;
+    height: 80vh;
+    transform: translate(-50%, -50%);
+    z-index: 100;
+  }
+
+  .pop-up-btn-box.desktop-only {
+    display: none;
+  }
+
+  .pop-up-close-btn {
+    margin-right: -16px;
+    height: 2rem;
+    border-radius: 3px;
+    padding: 0;
+    position: absolute;
+    top: 1.3%;
+    right: 4.6%;
+  }
+
+  .mobile-only .btn span {
+    font-size: 18px;
+  }
+
+  .pop-up-heading {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+
+  .pop-up-technologies button {
+    font-size: 1rem;
+    font-weight: 400;
+  }
+
+  .pop-up-description {
+    display: flex;
+    flex-direction: column;
+    gap: 0.6rem;
+  }
+
+  .pop-up-description p {
+    font-size: 1.2rem;
+    font-weight: 400;
+    line-height: 2rem;
+    text-align: left;
+  }
+}
+
+.pop-up {
+  scrollbar-gutter: stable;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  gap: 0.6rem;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  height: 75vh;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  background-color: #fff;
+  border-radius: 1rem;
+}
+
+.pop-up-img-box {
+  border-radius: 1.6rem;
+  position: relative;
+}
+
+.pop-up-hero-img {
+  width: 100%;
+}
+
+.pop-up-close-btn {
+  cursor: pointer;
+  background-color: #ebecf0;
+  float: right;
+}
+
+.pop-up-technologies {
+  display: flex;
+  gap: 8px;
+}
+
+.pop-up-technologies button {
+  border-radius: 4px;
+  padding: 8px 12px 8px 12px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.pop-up-btn-box {
+  display: flex;
+  gap: 2rem;
+  justify-content: flex-end;
+}
+
```

Comment:
I noticed that there is no picture nor button on the viewport of 768, kindly update it so we can have buttons to check the live and source code on every viewport.
<img width="555" alt="image" src="https://github.com/tajulafreen/Portfolio/assets/73854999/013f5572-dfe1-4b5c-aa01-8be01bf92141">